app/src/main/java/io/capibaras/abcall/data/repositories/IncidentsRepository.kt

```
@@ -65,4 +67,21 @@ class IncidentsRepository(
     }
+
+    suspend fun createIncident(name: String, description: String): Result<CreateIncidentResponse> {
+        return try {
+            val response = incidentsService.createIncident(CreateIncidentRequest(name, description))
+            if (response.isSuccessful) {
+                Result.success(response.body()!!)
+            } else {
+                val errorBodyString = response.errorBody()?.string()
+                handleErrorResponse(null, errorBodyString)
+            }
+        } catch (e: IOException) {
```

detekt

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)